src/xccl/ProcessGroupXCCL.cpp

```diff
@@ -62,6 +109,10 @@ ccl::reduction getXcclReduceOp(const ReduceOp& reduceOp, at::Tensor& input) {
       // Map sum to max for bool tensors to avoid overflow issues with sum.
       return ccl::reduction::max;
     }
+    // WA due to oneCCL not support AVG
+    if (reduceOp == ReduceOp::AVG) {
```

Comment:
The WA does not mean simply replacing `avg` with `sum`, but using `sum` collective and `div` SYCL kernel to simulate `avg`. Please update your comment.

Reply:
modified

Reply:
Please also add comments that `oneCCL is expected to support avg in basekit 2025.2 release.`

Reply:
done